ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueSubmission.scala

```diff
@@ -98,8 +100,11 @@ object KeyValueSubmission {
       displayName: Option[String],
       participantId: ParticipantId): DamlSubmission = {
     val party = hint.getOrElse("")
+    val inputDamlState =
+      partyAllocationDedupKey(participantId, submissionId) ::
```

Comment:
easier to read as: `List(x, y)`, or even preferably to avoid building the intermediate list you can just call `addInputDamlState` twice.

Reply:
Done